mlflow/genai/labeling/__init__.py

```diff
@@ -0,0 +1,26 @@
+"""
```

Comment:
I think we're just doing a vanilla wrapping here -- at the bottom of the doc on the labeling session change there is one important thing about doing anonymous labeling schemas. The way we'll have to do this is namespace the labeling schemas with the labeling session when registering it (so we can potentially have a lot of labeling schemas).

cc @dsmilkov